dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-3.0/io.openliberty.jaxb-3.0.feature

```diff
@@ -0,0 +1,33 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jaxb-3.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-API-Package: \
+  jakarta.activation; type="spec", \
+  jakarta.xml.bind;  type="spec", \
+  jakarta.xml.bind.annotation;  type="spec", \
+  jakarta.xml.bind.annotation.adapters;  type="spec", \
+  jakarta.xml.bind.attachment;  type="spec", \
+  jakarta.xml.bind.helpers;  type="spec", \
+  jakarta.xml.bind.util;  type="spec"
+IBM-ShortName: jaxb-3.0
+IBM-Process-Types: client, \
+ server
+Subsystem-Name: Jakarta XML Bindings 3.0
+-features=\
+  com.ibm.websphere.appserver.classloading-1.0, \
+  com.ibm.websphere.appserver.eeCompatible-9.0, \
+  io.openliberty.jakarta.jaxb-3.0
+-bundles=\
+  io.openliberty.jaxb.3.0.internal.tools
+-files=\
+  bin/jaxb/xjc.bat, \
+  bin/jaxb/tools/ws-schemagen.jakarta.jar, \
```

Comment:
I guess I am still struggling with how this jar file gets created.  Also how there is a Java EE version and a Jakarta EE version.  How does the scripts know which one to use.  Didn't see any script stuff.  So part of why I am so confused.